front/src/components/mmt-waypoint-transfer-card/mmt-waypoint-transfer-card.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="path-info">
+  <div class="vertical-line"></div>
+  <v-icon class="transport-icon" :class="{'rotate-180': transfer.transport === 0}">{{iconMap[transfer.transport]}}
+  </v-icon>
```

Comment:
English: you can specify a color for the icon. I don't really understand why to do this with a CSS variable if we already have this color defined in vuetify. I suggest just putting <v-icon color= "success base"/> and removing the CSS variable.